internal/args/unmarshal_test.go

```diff
@@ -99,15 +99,15 @@ func TestUnmarshalStruct(t *testing.T) {
 			"unknown-field=12",
 		},
 		expected: &Basic{},
-		error:    "unknown argument 'unknown-field'",
+		error:    "unknown argument with name 'unknown-field'",
```

Comment:
I am not entirely convinced `with name` provides any information since `unknown-field` is called an argument